maxim/logger/google_adk/client.py

```
@@ -265,21 +348,30 @@ async def after_model_callback(
+                        tool_span = parent_context.tool_call({
+                            "id": tool_call_id,
+                            "name": tool_name,
+                            "description": f"Tool call to {tool_name}",
+                            "args": str(tool_args),
+                            "tags": {"tool_call_id": tool_call_id, "from_llm_response": "true"},
+                        })
+                        # Store for potential result update in after_tool_callback
+                        self._tool_calls[tool_call_id] = tool_span
+                        print(f"[MaximSDK] Created tool call span: {tool_call_id} for {tool_name}")
+                        scribe().info(f"[MaximSDK] Created tool call span: {tool_call_id} for {tool_name}")
+                    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)